Extend docs on RIS reader and writer (i.e. partly labeled RIS) (#976)
asreview/asreview: docs/source/features/pre_screening.rst

Partly Labeled Data
-------------------

A partly labeled dataset contains labeling decisions for part of the records,
and part of the data does not have labels. The goal is to use the labels as
prior knowledge to screen the unseen records. The labeled records will be
recognized by ASReview LAB and are used to train the first iteration of the
active learning model. This might be helpful if you switch from screening in
another tool to screening with ASReview, or when updating an existing

> systematic review with more recent publications. For more options, see the
> `discussion board <https://github.com/asreview/asreview/discussions/975>`_. 

Reviewer: Google style guide prefers page title of the link.

```suggestion
systematic review with more recent publications. For more options, see `How to collaborate with ASReview? <https://github.com/asreview/asreview/discussions/975>`_ on the ASReview discussion board. 
```

.. note::

  Merging labeled with unlabeled data should be done outside ASReview LAB, for
  example in reference manager software.

For tabular datasets (:ref:`e.g., CSV, XLSX <data-format>`), the dataset should
contain a column, called :ref:`label_included <column-names>` which is
filled with 1's or 0's for the records that are already screened
and is empty for the records that you still need to screen using ASReview.

For the RIS file format, the dataset is handled automatically. The label 

> (ASReview_relevant, ASReview_irrelevant, ASReview_not_seen) is stored under the

Reviewer: ```suggestion
(`ASReview_relevant`, `ASReview_irrelevant`, `ASReview_not_seen`) is stored under the
```

N1 (Notes) tag. If the N1 tag is missing, it will be created for each record

> after importing the dataset. You can find a representative demo dataset in our
> `GitHub repository <https://github.com/asreview/asreview/blob/master/tests/demo_data/baseline_tag-notes_labels.ris>`_

Reviewer: ```suggestion
after importing the dataset. An example of a RIS file with N1 tag in the `ASReview
GitHub repository <https://github.com/asreview/asreview/blob/master/tests/demo_data/baseline_tag-notes_labels.ris>`_
```
